driver/tests/mpi/test_restart.py

```diff
@@ -81,4 +81,4 @@ def test_restart():
                     restart_physics[var].values,
                 )
     finally:
-        shutil.rmtree("RESTART")
+        shutil.rmtree("RESTART", ignore_errors=True)
```

Comment:
try-finally should have an `ignore_errors=True`. Again, unrelated to this PR but a tiny change we need to push though.